src/main/java/duke/Task.java

```diff
@@ -0,0 +1,61 @@
+package duke;
```

Comment:
Perhaps different types of tasks could be packaged together and named duke.tasks?